### PR TITLE
Adds support for multi-architecture builds (to support ARM for Apple …

### PR DESCRIPTION
**Pull request by dbernstein**
…M1 processors).

After this is merged,  https://github.com/fcrepo/fcrepo/pull/1923 can be merged.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#!/bin/bash
+
+# Builds a local docker image
+
```
Comment (whikloj) [inline build.sh]: I think `build.sh` is no longer used. It appears that `build-and-push-to-dockerhub.sh` is used instead...but I could be wrong.

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +1,24 @@
 jobs:
   include:
-    - if: branch = master
+    - if: branch = main
       env: VERSION=6.0.0-SNAPSHOT
-    - if: branch = 5.x-maintenance
-      env: VERSION=5.1.0
```
Comment (tomcbe) [inline .travis.yml:6]: Shouldn't we keep the configuration for 5.x versions with 5.1.2 on the way?

Comment (dbernstein) [reply .travis.yml:6]: actually - we don't want that because `main` supports 6.0.0 only.  There's a separate branch for 5.x-maintenance.

Comment (tomcbe) [reply .travis.yml:6]: Ah, so the pipeline definition in `main` is not relevant for th 5.x-maintenance branch?

**Review by tomcbe — APPROVED** [review]
Hey @dbernstein 

A side effect of combining the two scripts: it's no longer possible to build the docker image locally with the help of a script. Is that ok for us? Developers would then need to extract the script manually before beeing able to build. But I think thats ok.

File: .travis.yml (hunk 2 above)
Comment (dbernstein) [reply .travis.yml:6]: that's right.

**Comment by dbernstein** [comment]
@tomcbe:  Anyone wanting to build locally I expect would know enough about docker to know how to use the docker file.

**Pull request merged**